### ISSUE TITLE
Suggest SIGs posting their other materials as posts

### ISSUE THREAD

**Issue by whs-dot-hk** [kubernetes/community#2291]
Currently, each SIGs have their own way of posting additional files.

There might be different folders under a SIG, which made them hard to follow.

I suggest make those additional documents as post.

So, everyone could navigate them later as maybe News under a website.

**Comment by cblecker** [comment]
I'm confused. Can you clarify what you mean by "posts"?

**Comment by castrojo** [comment]
I think this is for hugo frontmatter at the beginning of each page so it knows how to render the page, similar to how docs does stuff like this: https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/administer-federation/daemonset.md

This would require some buy in and discussions first though.

**Comment by whs-dot-hk** [comment]
@cblecker The best we could do right now is

https://whs-dot-hk.github.io/k8sC-sigs-markdowns/overview/

Will link to the markdowns later

I think common tags could help

**Comment by nikhita** [comment]
> I suggest make those additional documents as post.

@whs-dot-hk I'm still confused. Can you elaborate on what you mean by "posts"?

**Comment by idealhack** [comment]
By reading at https://github.com/kubernetes/community/issues/2291#issuecomment-399161489, I guess @whs-dot-hk wanted better reading experiences for all the docs.

I think this can be useful, but I don't want to spend too much effort on it.

Also, I guess a website like this has some overlaps with our [contributor site](https://github.com/kubernetes/enhancements/blob/master/keps/sig-contributor-experience/0005-contributor-site.md).
